### PR TITLE
Add specific page web components and tests

### PR DESCRIPTION
**Pull request by TWpower**
- Add web components for specific page
- Add tests for those components

### CODE REVIEW

File: linku_frontend/linku/src/specific_page/RightPanel.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        this.state = {
+            place: '홍대',
+            time: "2017-02-25T18:00:00+09:00",
+            num_of_joined_appliers: 6,
+            max_num_ob_members: 6,
+            price : 10000
+        };
+    }
+
+    getTimeInStateTime(){
```
Comment (wkdtjsgur100) [inline linku_frontend/linku/src/specific_page/RightPanel.js]: You have to bind this function in constructor. 
like this 
> this.getTimeInStateTime = this.getTimeInStateTime.bind(this)

File: linku_frontend/linku/__test__/Appliers.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (wkdtjsgur100) [inline linku_frontend/linku/__test__/Appliers.test.js]: I think that the test file should be written to understand the role of the component.

File: linku_frontend/linku/src/specific_page/Place.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class Place extends React.Component{
+    constructor(props){
+        super(props);
+        this.state={
```
Comment (wkdtjsgur100) [inline linku_frontend/linku/src/specific_page/Place.js]: All states have to change to props.

Comment (TWpower) [inline linku_frontend/linku/src/specific_page/Place.js]: Done!

File: linku_frontend/linku/__test__/Appliers.test.js (hunk 2 above)
Comment (TWpower) [inline linku_frontend/linku/__test__/Appliers.test.js]: Done!

**Pull request merged**